test/blackbox-tests/test-cases/cram/custom-build-dir.t

```diff
@@ -21,7 +21,11 @@ path
     $TESTCASE_ROOT/tmp/default/foo.t.corrected
     to foo.t.
   [1]
-  $ sed -E '/\(pid: [0-9]+\)/{s//(pid: ###)/; s/instance.*/.../g; q;}' foo.t
+  $ sed -E 's/[0-9]+\)/###)/g' foo.t
     $ echo "  $ echo bar" >bar.t
     $ dune runtest
-    Error: A running dune (pid: ###) ...
+    Warning:
+    Your build request is being forwarded to a running Dune instance (pid:
+    ###). Note that certain command line arguments may be ignored.
+    Error: RPC server not running.
```

Comment:
The behaviour observed in this test is, imo, correct, but the error messages aren't helping at all.
Since DUNE_BUILD_DIR is set, the `dune runtest` of line 13 creates a lockfile that the `dune runtest` of line 10 will see. The inner dune will then try to send an RPC request to the outer dune, but this one wasn't launched in watch mode, and hence doesn't listen to RPC requests at all. The "server not running" message is correct, but we should better explain to the user what's going on

Reply:
Solved by latest commit